core/whistle-1.0.0/src/wh_util.erl

```diff
@@ -21,8 +21,13 @@
          ,is_system_db/1
         ]).
 -export([get_account_realm/1, get_account_realm/2]).
--export([is_account_enabled/1]).
--export([is_account_expired/1]).
+-export([is_account_enabled/1, is_account_expired/1]).
+-export([maybe_disable_account/1
```

Comment:
All of these account document-related functions should be in `kz_account`; that was my point in previous comments. Since they all operate based on the account doc, they should be moved there instead of here. Slowly trying to clean up `wh_util`